ui/src/Env.elm

```diff
@@ -0,0 +1,12 @@
+-- Copyright (c) 2019
+-- Mainflux
+--
+-- SPDX-License-Identifier: Apache-2.0
+
+
+module Env exposing (env)
+
+
+env =
+    { url = ""
```

Comment:
Please use default value. If the user did not set ENV var, it should fallback to something like `http://localhost:3000`, right?

Reply:
Ahah, probably again - defult should be left empty, because of Docker deployment?